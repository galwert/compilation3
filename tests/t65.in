// should fail - symbol redeclared (fib)
int fib(int fib)
{
    if (fib == 0 or fib == 1) {
        return 1;
    }
    return fib(fib - 1) + fib(fib - 2);
}

void main() {     return fib(0); }
// because