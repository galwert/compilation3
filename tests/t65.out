line 2: identifier fib is already defined
